<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProject</class>
 <widget class="QDialog" name="CreateProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Aardwolf Project</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_ProjectName">
         <property name="text">
          <string>Project Name</string>
         </property>
         <property name="buddy">
          <cstring>projectName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="projectName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_ProjectLocation">
         <property name="text">
          <string>Project Directory</string>
         </property>
         <property name="buddy">
          <cstring>projectLocation</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="projectLocation">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="chooseFolder">
         <property name="toolTip">
          <string>Choose Location</string>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_ProjectInformation">
         <property name="text">
          <string>Project Information</string>
         </property>
         <property name="buddy">
          <cstring>projectInfo</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="storeInXMP">
         <property name="text">
          <string>Store Metadata in XMP</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_XMP_Note">
         <property name="text">
          <string>(Note: This may slow down annotations)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="2" colspan="2">
        <widget class="QPlainTextEdit" name="projectInfo"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="CancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CreateButton">
         <property name="text">
          <string>Create</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectName</tabstop>
  <tabstop>chooseFolder</tabstop>
  <tabstop>projectInfo</tabstop>
  <tabstop>projectLocation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
